include/linux/list.h

```diff
@@ -362,6 +362,17 @@ static inline void list_splice_tail_init(struct list_head *list,
 	list_entry((ptr)->next, type, member)
 
 /**
+ *  * list_last_entry - get the last element from a list
+ *   * @ptr:	the list head to take the element from.
+ *    * @type:	the type of the struct this is embedded in.
+ *     * @member:	the name of the list_struct within the struct.
+ *      *
+ *       * Note, that list is expected to be not empty.
+ *        */
```

Comment:
We aren't accepting pull requests for PMFS, as we have halted work on this project in favor of LIBNVDIMM and DAX.  For more information on this solution please see:
https://nvdimm.wiki.kernel.org/

Regarding this specific change, I do have a few items of feedback:

1) You're adding list_last_entry() which was copied from another implementation, but you never use it.  Typically we don't add code that is unused, so this code would need a consumer before being added.

2) When pasting the comment, you didn't properly tell your editor that you were pasting, hence the weird double comment lines where the editor automatically indented the next line for you and added a '*', and you also got spacing and a '*' from the pasted code:

 /**
 + *  * list_last_entry - get the last element from a list
 + *   * @ptr:	the list head to take the element from.
 + *    * @type:	the type of the struct this is embedded in.
 + *     * @member:	the name of the list_struct within the struct.
 + *      *
 + *       * Note, that list is expected to be not empty.
 + *        */

In vim you can get around this by putting your editor in "paste" mode with ":set paste".  I'm sure other editors like emacs have an equivalent paste mode.

3) In Linux development your git commit needs to have a title, a descriptive commit message, and a signed-off-by line.  If you're interested in getting involved in kernel development I"d suggest starting here:

https://kernelnewbies.org/

Thanks,
- Ross